spring-cloud-kubernetes-client-discovery/src/main/java/org/springframework/cloud/kubernetes/client/discovery/KubernetesInformerDiscoveryClient.java

```diff
@@ -55,7 +55,7 @@
  * @author Ryan Baxter
  * @author Tim Yysewyn
  */
-public class KubernetesInformerDiscoveryClient implements DiscoveryClient, InitializingBean {
```

Comment:
I would like to continue cleaning up some classes via small PRs, so that they are easy to review. This one does the obvious thing of replacing `InitializingBean` with `@PostConstruct`